classes/manage_content_types.php

```diff
@@ -79,6 +79,14 @@ public function execute(string $action): void {
                 $this->delete(required_param('id', PARAM_INT));
                 redirect(new \moodle_url(self::get_base_url()));
                 break;
+            case 'show':
+                $this->set_visibility(required_param('id', PARAM_INT), true);
+                notification::success(get_string('visibility_updated', 'mod_cms'));
```

Comment:
a super minor thing, queueing a notification and the redirecting is so common that this is part of the redirect api and you can merge these two lines

Reply:
done